docs/theming/index.md

```diff
@@ -17,22 +18,23 @@ order: 1
   import '@rhds/elements/rh-tile/rh-tile.js';
 </script>
 
-Red Hat's Design System comes with a powerful, flexible, and easy-to-use
-theming system. Design system elements come with branded, accessible defaults 
-built-in, so that all you need to do to create quintessentially Red Hat 
-experiences is write some HTML. For those occasions when we want to flex our 
-design muscles, our elements come with powerful theming primitives in the form 
-of **slots**, **design tokens**, and **CSS shadow parts** to enable you to theme 
-a single element, a section, a page, or an entire app.
+## Overview
```

Comment:
The h1 is also "Overview". I know the design has "Overview" 2x. Is it necessary? YMMV.

Reply:
I debated the same. @coreyvickery please advise.

Reply:
Removed.